src/libcore/slice/mod.rs

```diff
@@ -1607,6 +1611,10 @@ impl<T> [T] {
     /// let src = [1, 2, 3, 4];
     /// let mut dst = [0, 0];
     ///
+    /// // Note: the slices must be the same length, so
```

Comment:
Nit: The line breaks look somewhat strange here, the first two lines break at a different width than the last two lines.